controllers/dpa_controller.go

```diff
@@ -146,16 +148,53 @@ func (r *DPAReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.Service{}).
 		Owns(&routev1.Route{}).
 		Owns(&corev1.ConfigMap{}).
-		Watches(&source.Kind{Type: &corev1.Secret{}}, &labelHandler{}).
+		Watches(&source.Kind{Type: &corev1.Secret{}}, &secretHandler{}).
 		WithEventFilter(veleroPredicate(r.Scheme)).
 		Complete(r)
 }
 
-type labelHandler struct {
+type secretHandler struct {
+	defauldSecrets map[string]*struct{}
```

Comment:
Gotta love this guy.
```suggestion
	defaultSecrets map[string]*struct{}
```